db.js

```diff
@@ -863,6 +913,9 @@ exports.init = function (sbot, config) {
     reindexEncrypted,
     indexingProgress: () => indexingProgress.listen(),
 
+    buttPost,
```

Comment:
Can you help me understand what is `buttPost` for? Why isn't `post` enough? (I'm thinking and planning whether we need `post` for each feed format)

Reply:
Post worked for bendy butt because we made it behave very similar to a normal msgval. If we really want to do binary formats, then we need something where we can give the raw bytes. So i added this. I was thinking we should have one post per feed type. Then in EBT you don't need to test so much. You already know the format.

Reply:
Not really one variable per type. Maybe a map

Reply:
Okay, thanks for the context. I have a few options how to take that into account. I'm thinking of renaming (or maybe not renaming) `post` to `onAdded` or `onMsgAdded` and then besides passing the msg (bytes or object), it would also tell what format it was. That would inform EBT.